src/main/kotlin/org/eclipse/tractusx/bpdmcertificatemanagement/entity/EnclosedSiteDB.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.eclipse.tractusx.bpdmcertificatemanagement.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Embeddable
+
+@Embeddable
+data class EnclosedSiteDB(
+
+    @Column(name = "site_bpn", nullable = false)
+    val siteBpn: String,
+
+    @Column(name = "area_of_application")
```

Comment:
nullable = false? String type is at least not nullable

Reply:
Yeah done.